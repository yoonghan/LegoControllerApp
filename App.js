import React from 'react';
import {Provider} from "react-redux";
import store from "./src/redux/reducer";
import Main from "./src/screen/Main";

const App: () => React$Node = () => {
  return (
    <Provider store={store}>
      <Main/>
    </Provider>
  );
};

export default App;
