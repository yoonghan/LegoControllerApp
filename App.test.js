import React from 'react';
import {shallow, mount} from 'enzyme';
import App from './App';

describe('App', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<App/>)
            expect(component).toMatchSnapshot();
        });
        it('component can be tested', () => {
            const wrapper = mount(<App/>)
            expect(wrapper.findWhere(node => node.prop('testID') === 'jest-test')).toBe(true);
        });
    });
});
