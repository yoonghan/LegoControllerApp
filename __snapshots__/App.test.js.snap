// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
